_posts/2018-02-12-continuous-releases.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+tags: development-process
+date: 2018-02-12 13:14
+title: The Hypothesis continuous release process
+published: true
+author: alexwlchan
+---
+
+If you watch [the Hypothesis changelog][changelog], you'll notice the rate of releases sped up dramatically in 2017.
+We released over a hundred different versions, sometimes multiple times a day.
+
+This is all thanks to our continuous release process.
+We've completely automated the process of releasing, so every pull request that changes code gets a new release, without any human input.
+In this post, I'll explain how our continuous releases work, and why we find it so useful.
+
+[changelog]: https://hypothesis.readthedocs.io/en/latest/changes.html
```

Comment:
Huh. I didn't know about this markdown link syntax.

Reply:
Yeah, it's quite nice for including links without cluttering up your prose with URLs.